.github/workflows/labeler.yml

```diff
@@ -0,0 +1,14 @@
+name: "Pull Request Labeler"
```

Comment:
Is this duplicate with this PR?
https://github.com/layer5labs/kanvas-snapshot/pull/118/files

Reply:
I wasn't aware of this pull request.
I have added the [labeler workflow](https://github.com/layer5labs/kanvas-snapshot/pull/121/files#diff-09b72f3c9a3e4f00ab00cd7000b302db25f056075d8895bd91b3654d6e7e956b) as well as the [labels](https://github.com/layer5labs/kanvas-snapshot/pull/121/files#diff-a22c263686553013feaeb0677d26eeb0b8778a756c4311c1fce13384258026aa) here.